app/filtering.js

```diff
@@ -372,7 +367,7 @@ function registerPermissionHandler (session, partition) {
       cb(result)
       if (persist) {
         // remember site setting for this host over http(s)
-        appActions.changeSiteSetting(urlOrigin, permission + 'Permission', result, isPrivate)
+        appActions.changeSiteSetting(getOrigin(url), permission + 'Permission', result, isPrivate)
```

Comment:
this should be `origin` instead of getOrigin(url) since the message shown to the user was for `origin`